internal/models/policy/recipe/custom/test_host_networking.go

```diff
@@ -0,0 +1,90 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+Code generated by go-swagger; DO NOT EDIT.
+*/
+
+package policyrecipecustommodel
+
+import (
+	"github.com/go-openapi/swag"
+
+	policyrecipecustomcommonmodel "github.com/vmware/terraform-provider-tanzu-mission-control/internal/models/policy/recipe/custom/common"
+)
+
+// VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TestHostNetworking test-host-networking recipe schema.
+//
+// The input schema for test-host-networking recipe.
+//
+// swagger:model VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TestHostNetworking
+type VmwareTanzuManageV1alpha1CommonPolicySpecCustomV1TestHostNetworking struct {
```

Comment:
should the file name start with test ? this is not test file right ?

Reply:
that's the name of the recipe `test-host-networking`

Reply:
Realised that it was a user made recipe, hence removed it.